Nns spawn neuron to disburse (#5845)
dfinity/portal: docs/building-apps/governing-apps/nns/using-the-nns-dapp/nns-dapp-advanced-neuron-operations.mdx

---

import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
import "/src/components/CenterImages/center.scss";

# Advanced neuron operations

<MarkdownChipRow labels={["Intermediate", "Governance", "Tutorial"]} />

Once you're more familiar with the NNS, you might want to explore additional
neuron operations using the NNS dapp.

For more background on neurons—including voting rewards and detailed
explanations of available operations—refer to the
[Neurons overview](https://learn.internetcomputer.org/hc/en-us/articles/34084120668692-NNS-Neurons).

> ## Maturity Disbursement

Reviewer: ```suggestion
## Maturity disbursement
```

As you vote on proposals (either directly or by following other neurons), your
neuron's maturity increases. This maturity represents your accumulated
governance rewards.

When you have at least 1.05 of maturity that is not staked, you can disburse it.
This operation burns the maturity and schedules the minting of new ICP tokens.
After 7 days, newly minted ICP will be minted to the selected destination
address.

> > **Note:** For Ledger devices, [Spawn Neuron](#spawn-new-neurons) is available.
> > Support for **Disburse Maturity** will be added later.

Reviewer: ```suggestion
:::info
For Ledger devices, [spawn neuron](#spawn-new-neurons) is available. Support for **disburse maturity** will be added in the future.
:::
```
